File: repos_estudiantes_TARQUI_segunda_entrega/RUBIN_DE_CELIS_VALDEZ_ANDRES/COMPOSICION/EJERCICIO-1/casa.py
from habitacion import Habitacion

class Casa:
    def __init__(self, direccion):
        self.direccion = direccion
        self.habitaciones = []

    def get_direccion(self):
        return self.direccion

    def get_habitaciones(self):
        return self.habitaciones

    def set_direccion(self, direccion):
        self.direccion = direccion

    def agregar_habitacion(self, habitacion):
        self.habitaciones.append(habitacion)

    def mostrar_casa(self):
        print(f"Direccion: {self.direccion}")
        print("Habitaciones:")
        for hab in self.habitaciones:
            hab.mostrar_info()

File: repos_estudiantes_ALVAREZ_segunda_entrega/GUTIERREZ_GARCIA_CARLA_ALEJANDRA/COMPOSICION_AGREGACION/Ejercicio1/Casa.java
import java.util.ArrayList;
import java.util.List;

public class Casa {
    private String direccion;
    private List<Habitacion> habitaciones;

    public Casa(String d) {
        this.direccion = d;
        this.habitaciones = new ArrayList<>();
    }

    public String getDireccion() {
        return direccion;
    }

    public List<Habitacion> getHabitaciones() {
        return habitaciones;
    }

    public void setDireccion(String d) {
        this.direccion = d;
    }

    public void agregar_habitacion(Habitacion habitacion) {
        habitaciones.add(habitacion);
    }

    public void mostrar_casa() {
        System.out.println("Direccion de la casa: " + this.direccion);
        System.out.println("Habitaciones:");
        for (int i = 0; i < habitaciones.size(); i++) {
            habitaciones.get(i).mostrar_info();
        }
    }
}